rust/Cargo.toml

```diff
@@ -11,7 +11,7 @@ categories  = ["cryptography", "no-std"]
 keywords    = ["cryptography", "encryption", "security", "streaming"]
 
 [dependencies]
-aesni = "0.1"
+aesni = { git = "https://github.com/RustCrypto/block-ciphers.git" }
```

Comment:
Note: using an unreleased version of aesni. We should wait for this to be released before merging.